crates/client/rpc/src/versions/admin/v0_1_0/mod.rs

```diff
@@ -1,4 +1,10 @@
 pub mod api;
 pub mod methods;
 
+use std::time::SystemTime;
+
 pub use api::*;
+
+fn unix_now() -> u64 {
```

Comment:
why is it here

Reply:
I think I hoisted this after some changes to the admin rpcs to make it available across multiple files, but I don't think that's necessary anymore.

Reply:
This has been fixed. Hoisted down to the relevant file.